region/brinstar/green/Green Brinstar Main Shaft.json

```diff
@@ -4706,6 +4706,26 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [10, 9],
+      "name": "Use Flash Suit",
+      "requires": [
+        {"useFlashSuit": {}},
```

Comment:
gain/use flashSuit have a kind of canShinechargeMovementComplex/Tricky.  Its kinda undefined how it works; its like a flash suit equivalent.

Reply:
I think those are more focused on moving with a tight shinecharge timer.